spydrnet_physical/util/tests/test_Connection_Pattern.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+from spydrnet_physical.util import ConnectPoint , ConnectPointList, ConnectionPattern
+
+
+class test_Connection_Pattern(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.width = 11
+        self.height = 11
+        self.conn_manager = ConnectionPattern(self.width, self.height)
+        self.conn_patt = self.conn_manager.connections
+        self.conn_patt.cursor = (6, 0)
+        self.conn_patt.move_y(steps=6)
+
+    def test_get_htree(self):
```

Comment:
you need to also check what happens when the parameter passed to h-tree is not 2^n +1